crates/rethnet_rpc_server/Cargo.toml

```diff
@@ -4,10 +4,16 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
+axum = "0.6.18"
 hashbrown = { version = "0.13", default-features = false, features = ["ahash"] }
+hyper = "0.14.26"
 rethnet_eth = { version = "0.1.0-dev", path = "../rethnet_eth", features = ["serde"] }
+rethnet_evm = { version = "0.1.0-dev", path = "../rethnet_evm" }
+rethnet_evm_napi = { version = "0.1.0-dev", path = "../rethnet_evm_napi" }
+revm = { git = "https://github.com/Wodann/revm", rev = "a4550cc", version = "3.1", default-features = false, features = ["dev", "secp256k1", "serde", "std"] }
```

Comment:
What do we need `revm` for?

Reply:
i was using it in a few places but it was unnecessary and i eliminated those references (and this dependency) in https://github.com/NomicFoundation/hardhat/pull/3998/commits/6c036a9c5c614a8f227d2a8ecaebc10bc73cab78